CI fails to push artifacts to the flatpak-api server

### What Happened?

Latest attempt of Daily actions (for commit 67fbb1c367ef299a23022c9182223457c169eec1) is failing, both x86 and ARM are reporting the same error:

- [Publish Platform (x86)](https://github.com/elementary/flatpak-platform/actions/runs/25678136416/job/75381810744)
- [Publish Platform (ARM)](https://github.com/elementary/flatpak-platform/actions/runs/25678136416/job/75381810852)

```
Run flat-manager-client push https://flatpak-api.elementary.io/api/v1/build/12236 ./elementary
  flat-manager-client push https://flatpak-api.elementary.io/api/v1/build/12236 ./elementary
  shell: /usr/bin/bash -e {0}
  env:
    build_id: https://flatpak-api.elementary.io/api/v1/build/12236
    REPO_TOKEN: ***
Uploading refs to https://flatpak-api.elementary.io/api/v1/build/12236: runtime/io.elementary.Platform.Locale/aarch64/daily, runtime/io.elementary.Platform/aarch64/daily, runtime/io.elementary.Sdk.Debug/aarch64/daily, runtime/io.elementary.Sdk.Locale/aarch64/daily, runtime/io.elementary.Sdk/aarch64/daily
Refs contain 16625 metadata objects
Remote missing 1326 of those
Has 210713 file objects for those
Remote missing 42053 of those
Uploading file objects
Uploading 671 files (4192935 bytes)
Uploading 378 files (2746765 bytes)
Uploading 325 files (4042151 bytes)
Uploading 468 files (4185777 bytes)
Uploading 727 files (4041382 bytes)
Uploading 770 files (3984326 bytes)
Uploading 97 files (3995012 bytes)
Uploading 504 files (4190558 bytes)
Uploading 643 files (4185902 bytes)
Uploading 628 files (4193957 bytes)
Uploading 821 files (4193614 bytes)
Uploading 202 files (1511163 bytes)
Uploading 88 files (3916057 bytes)
Uploading 570 files (3366758 bytes)
Uploading 234 files (3732221 bytes)
Uploading 803 files (4194085 bytes)
Uploading 712 files (4160075 bytes)
Uploading 472 files (4183121 bytes)
Uploading 240 files (1297963 bytes)
Uploading 193 files (4185077 bytes)
Uploading 1036 files (4140500 bytes)
Uploading 510 files (3307188 bytes)
Uploading 231 files (3986646 bytes)
Uploading 365 files (1731423 bytes)
Uploading 1 files (8803806 bytes)
Uploading 684 files (4193702 bytes)
Uploading 194 files (1579147 bytes)
Uploading 1 files (5360583 bytes)
Uploading 256 files (1300453 bytes)
Uploading 1 files (397125899 bytes)
API error from https://flatpak-api.elementary.io/api/v1/build/12236/upload (status 413): {"message":"Non-json error from server: <html>\r\n<head><title>413 Payload Too Large</title></head>\r\n<body>\r\n<center><h1>413 Payload Too Large</h1></center>\r\n<hr><center>cloudflare</center>\r\n</body>\r\n</html>\r\n"}
Error: Process completed with exit code 1.
```


### Steps to Reproduce

See the above results.

### Expected Behavior

Daily actions should succeed.


### OS Version

Other Linux

### OS Architecture

amd64 (on most hardwares)

### Session Type

Secure Session (Wayland, This is the default)

### Software Version

Early Access or Compiled from git

### Log Output

```shell

```

### Hardware Info

_No response_

--- a/.github/workflows/daily.yml
+++ b/.github/workflows/daily.yml
@@ -70,7 +70,7 @@ jobs:
         env:
           REPO_TOKEN: ${{ secrets.FLAT_MANAGER_TOKEN }}
         run: |
-          BUILD_ID=`flat-manager-client create https://flatpak-api.elementary.io appcenter`
+          BUILD_ID=`flat-manager-client create https://flatpak-api.elementaryos.org appcenter`
           echo "build_id=$BUILD_ID" >> $GITHUB_ENV
 
       - name: Push Build

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -73,7 +73,7 @@ jobs:
         env:
           REPO_TOKEN: ${{ secrets.FLAT_MANAGER_TOKEN }}
         run: |
-          BUILD_ID=`flat-manager-client create https://flatpak-api.elementary.io appcenter`
+          BUILD_ID=`flat-manager-client create https://flatpak-api.elementaryos.org appcenter`
           echo "build_id=$BUILD_ID" >> $GITHUB_ENV
 
       - name: Push Build

